maquetado/scss/detail.scss

```diff
@@ -0,0 +1,64 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+%equal-font-heights {
+ 
+  font-family: Roboto, sans-serif;;
```

Comment:
Since everything is going to have the same font, you should set it to every element like:

```css
* {
  font-family: Roboto, sans-serif;
}
```

(Check that double semicolon, delete one)